src/lib/operators/projection.cpp

```diff
@@ -213,6 +182,7 @@ std::shared_ptr<const Table> Projection::_on_execute() {
   Hyrise::get().scheduler()->schedule_and_wait_for_tasks(jobs);
```

Comment:
I don' think so. Basically, we have three cases for each column in the projection:
1. column is forwarded. `timer.lap()` is called and added to `forwarding_cost`. 
2. column needs evaluation and
   - (i) input is small. Directly evaluate expression, call `timer.lap()` and add to `expression_evaluator_cost`
   - (ii) input is large. Create task, do **not** call `timer.lap()`.

All remaining expressions from 2.(ii) are then scheduled and `timer.lap()` is added to `expression_evaluator_cost`. If neither 1. or 2.(ii) occured (all scheduled), this is just the duration of everything. If, at any point, 1. or 2.(i) occured, `timer.lap()` was called somewhen, and the call here is just the time elapsed since then.